web/src/lib/sources.ts

```diff
@@ -111,6 +111,12 @@ const SOURCE_METADATA_MAP: SourceMap = {
     category: SourceCategory.ProjectManagement,
     docs: "https://docs.danswer.dev/connectors/jira",
   },
+  jira_service_management: {
```

Comment:
I believe Jira Service Management has a different icon (see the icon [here](https://www.atlassian.com/software/jira/service-management?&aceid=&adposition=&adgroup=140639872753&campaign=17732529748&creative=639445349742&device=c&keyword=jira%20service%20management&matchtype=e&network=g&placement=&ds_kids=p72254545591&ds_e=GOOGLE&ds_eid=700000001721198&ds_e1=GOOGLE&gad_source=1&gclid=Cj0KCQjw_sq2BhCUARIsAIVqmQsY-qw1JjOv6Cx9A-pTQxfvXjlVprEkFySFVZc32heJFob7IdCPXQ0aAlJjEALw_wcB&gclsrc=aw.ds). Can we use that one instead?

Reply:
okay... i will get right on it

Reply:
@jaykayudo any updates here?

Reply:
@Weves I have changed the icon and i have added a unit test. you can see that by reviewing the file changes. But about the video, i have been having issues installing some of the dependencies for danswer so i have not been able to do it. Please is there a docker compose for everything that makes it easier to run?

Reply:
@jaykayudo yes! Have you checked out our quickstart guide: https://docs.danswer.dev/quickstart. 

This should get things up and running locally!